go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -590,3 +661,106 @@ func getClusterSubnetsExclusion() string {
 	}
 	return exclusion
 }
+
+// AddNodeDNSInfo is called when a NodeDNSInfo object is created, resposible for creating initial data
+// for dnsNames resolved by nodes
+func (efDNS *egressfirewallDNSInformation) AddNodeDNSInfo(nodeDNSInfo *nodednsinfoapi.NodeDNSInfo) error {
```

Comment:
I am not a fan of the naming convention, one is suffixed with "Information" and the other "Info". Could we align it?